tests/unit/test_self_check_outdated.py

```diff
@@ -6,7 +6,7 @@
 from typing import Any, Optional, cast
 from unittest import mock
 
-import freezegun  # type: ignore
+import freezegun  # type: ignore  # mypy: no-warn-unused-ignores
```

Comment:
Why do we need this type: ignore? What fails if we remove it?

Reply:
It was needed for me at some point but I just tried and this is not needed anymore. Not sure what changed, but I’m only interested in getting my pre-commit to work TBH so let’s revert this.